### DISCUSSION TITLE
Why does not 'hidden' apply on my div?

### DISCUSSION THREAD

**Discussion by arnelamo** [tailwindlabs/tailwindcss#9885] [Help] [answered]
Hi, everyone. I'm new to Tailwind, so please bear with me. 

I got a **div** with the following styles:

`
        <div
          className="items-center justify-between w-full md:flex md:w-auto md:order-1 hidden"
          id="TEST"
        >
          <ul>
          // ... content
         </ul>
       </div>
`

What I don't understand is this:
in my world, this div and it's content should always be hidden (display: none;). In stead, from breakpoint `md` and up, the `ul` is actually displayed. 

can anyone explain to me _why_ is this happening?

Screenshot of computed styles on the div:
<img width="1000" alt="Screenshot 2022-11-20 at 19 26 47" src="https://user-images.githubusercontent.com/48482148/202919319-69d2c5ed-c2ff-4ba8-9ea6-44c4c66fac57.png">

**Comment by wongjn** [comment, marked as answer]
Yes, I can explain. You have `hidden` class which applies `display: none`, but this is overridden by the `md:flex` class that applies `display: flex`. This is because Tailwind organizes class names such that class names with variants appear after class names without and order of rules in the stylesheet determine which rule takes precedence when the rules have the same specificity.

**Reply by arnelamo** [reply]
Now it makes sense.

Thanks
